README-nl.md

```diff
@@ -10,64 +10,64 @@ Minecraft Console Client (MCC) is een simpele app waarmee je Minecraft servers k
 Krijg het binary bestand van onze laatste [development build](https://github.com/MCCTeam/Minecraft-Console-Client/releases/latest).
 De exe file is een .NET binary die ook werkt op Mac en Linux.
 
-## How to use 📚
+## Hoe Te Gebruiken  📚
 
 Bekijk de [sample configuration files](MinecraftClient/config/) die de how-to-use bevat [README](https://github.com/MCCTeam/Minecraft-Console-Client/tree/master/MinecraftClient/config#minecraft-console-client-user-manual).
 
-## Getting Help 🙋
+## Hulp Krijgen 🙋
 
 Bekijk de [README](https://github.com/MCCTeam/Minecraft-Console-Client/tree/master/MinecraftClient/config#minecraft-console-client-user-manual) en de bestaande [Discussions](https://github.com/MCCTeam/Minecraft-Console-Client/discussions): Misschien is je vraag daar al beantwoord. Als dat niet zo is open dan een [New Discussion](https://github.com/MCCTeam/Minecraft-Console-Client/discussions/new) en stel je vraag. Als je een bug vind, raporteer dat in de [Issues](https://github.com/MCCTeam/Minecraft-Console-Client/issues) sectie.
 
-## Helping Us ❤️
+## Ons helpen ❤️
```

Comment:
"Steentje bijdragen" sounds better, this isnt wrong tho

Reply:
Sounds allot better I will change it.